src/main/src/main/java/org/geoserver/catalog/ResourceInfo.java

```diff
@@ -374,4 +374,16 @@
 
     /** Set the list of disabled services names for this resource */
     void setDisabledServices(List<String> disabledServices);
+
+    /**
+     * Flag that determines if complex features will be converted to simple feature for compatible
+     * output formats.
+     */
+    Boolean getSimpleConversionEnabled();
```

Comment:
As per my previous comment, this is a boolean properly, should be using "is" instead of "get":

```suggestion
    Boolean isSimpleConversionEnabled();
```

Reply:
Understood, fixed.